lib/kennel/compatibility.rb

```diff
@@ -4,8 +4,11 @@ module Kennel
   module Compatibility
     def self.included(into)
       class << into
-        %I[out out= err err= strict_imports strict_imports= generate plan update].each do |sym|
-          define_method(sym) { |*args| instance.public_send(sym, *args) }
+        %I[generate plan update].each do |sym|
```

Comment:
FYI actually prefer the simpler:
```suggestion
        [:generate, :plan, :update].each do |sym|
```